TACC/Core-Portal
# Task/TV3-105: Historical jobs in V3

## Overview
Add temporary solution to support historical v2 jobs in v3.

## Related

* [TV3-93](https://jira.tacc.utexas.edu/browse/TV3-93)
* [TV3-105](https://jira.tacc.utexas.edu/browse/TV3-105)

## UI
![Screen Shot 2022-12-15 at 12 05 17 PM](https://user-images.githubusercontent.com/9425579/207944184-302eef39-dcb2-4583-b370-ec86216e6e63.png)

## Changes
- [x] Migration script with raw tapis v2 result in `JobSubmission` model.
- [X] Separate route for v2 jobs
- [X] Sagas/reducers
- [X] New sidebar section in history view
- [X] ~New job listing with warning message on top (I think this could be done through custom messages since this is an ephemeral feature)~ (Should do through custommessages)
- [X] v2 job modal with omitted information (e.g. resubmit)
- [X] Backport Frontend processing v2 jobs
- [x] Backport some v2 tests

## Testing Steps
1. Create a dummy v2 jobsubmission model mimicking what's created with #748 
2. Ensure that the historic job listing works (https://cep.test/workbench/history/jobsv2)
3. Ensure that the jobs details modal also renders properly.

## Notes
I wasn't sure how we should go about the warning message.
Using the section messages functionality seems to be difficult with smaller changes. I was thinking maybe we could use each portal's custom message functionality to display the ephemeral message.
